### DISCUSSION TITLE
Mimic "created_by" column in Postgresql database

### DISCUSSION THREAD

**Discussion by bmscmoreira** [Budibase/budibase#12305] [Help] [answered]
I have a Postgresql table linked in Budibase and I would like, for each inserted row, to have a specific column (eg. "author") to be updated with the currently logged in user in Budibase. Can someone point me in the right direction on how can I achieve this? Thank you.

**Comment by sdsys-ch** [comment, marked as answer]
Hi @bmscmoreira , probably a couple of ways to do that.

I have `created_by` and `updated_by` rows by default in all relevant postgres tables.

You can use  any of the [`Current User` bindings](https://docs.budibase.com/docs/users-table#current-user-bindings), e.g. `{{ Current User._id }}` as the default value of a form field, disable the field if you want and/ or hide it with css (`visibility: hidden;` in the Styles' `Edit Custom CSS` setting). Works like a charm. 

However, don't use a conditional to hide the element since it would be [completely unmounted](https://github.com/Budibase/budibase/issues/7247#issuecomment-1214170992) and will then either throw an error for any required db field or not update the field for all others.

If immutability or uniqueness for the author/... value is a must (the above solution isn't safe but safe enough for my use case), re-/set the value in an action before the `Save Row` action is triggered.

There's also the [`User`-type column field](https://docs.budibase.com/docs/users-1) that works with sql tables in budibase but haven't used it for this. 

Also, if you need a more complex setup, you may want to ["re-create" budibase's User table in your postgres db](https://github.com/Budibase/budibase/discussions/5847#discussioncomment-5025369).

Hope this helps!
